src/test/mir-opt/matches_reduce_branches.match_nested_if.MatchBranchSimplification.64bit.diff

```diff
@@ -4,80 +4,36 @@
   fn match_nested_if() -> bool {
       let mut _0: bool;                    // return place in scope 0 at $DIR/matches_reduce_branches.rs:39:25: 39:29
       let _1: bool;                        // in scope 0 at $DIR/matches_reduce_branches.rs:40:9: 40:12
-      let mut _2: bool;                    // in scope 0 at $DIR/matches_reduce_branches.rs:41:18: 41:76
```

Comment:
This test does not test `MatchBranchSimplification` any more, but should provides a good test for `const_goto`.
Could you:
- copy it into a test in `const_goto.rs`?
- make this test more robust by replacing some occurrences of `true` by function parameters to prevent const_goto from triggering?